quartz_solar_forecast/forecasts/v2.py

```diff
@@ -62,29 +65,27 @@ def _decompress_zipfile(self, filename: str) -> None:
     def load_model(
         self, 
         model_file: str = constants.MODEL_FILE,
-        file_id: str = constants.FILE_ID
+        repo_id: str = "openclimatefix/open-source-quartz-solar-forecast",
+        file_path: str = "models/v2/model_10_202405.ubj.zip"
     ) -> XGBRegressor:
-        """
-        Download and decompress model from Google Drive
-        Parameters
-        ----------
-        model_file: str
-            The name of the model as string
-        file_id: str
-            Google id to download the file
-        """
-        zipfile_model = model_file + ".zip"
-
```

Comment:
Can you add function descriptions here and the other function

Reply:
> Can you add function descriptions here and the other function

yeah sure